_posts/2023-11-02-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.44
+author: Priya
+date: 2023-11-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Date Difference Calculation - Now with System Time and Specific Date Options:
+Calculate date differences more effectively with additional choices. You can choose between system time, specific dates, and even set calculations to the current date-time in UTC. The date difference component is also conveniently displayed in pipelines (Years, Months, Weeks, etc).
+
+#### Insert Current Date and Time - Now with UTC Option:
+Creating a column and inserting the current date and time is easier than ever. You can now choose to insert the current date and time in UTC using the label and insert value rule, giving you more control over your data.
+
+#### Improved Folder Deletion Handling for Dependency Issues:
+We've enhanced the folder deletion process to ensure consistent behavior with deletion operations. Now, items inside the folder are deleted first, and only after that, the folder is removed. This prevents issues with dependencies during dataset (ds) deletion.
+
+#### Improved User Experience During Instance Wake-Up:
```

Comment:
#### Signing in to a workspace after a break will show a instance warm up information box
When a User is away and there is no activity ina workspace the underlying compute is suspended. We now show an information modal when user activates a workspace. The modal informs that the workspace is getting ready.